Modules/Devices/src/Devices.Infrastructure/Persistence/Database/EntityConfigurations/IdentityEntityTypeConfiguration.cs

```diff
@@ -9,7 +10,7 @@ public class IdentityEntityTypeConfiguration : IEntityTypeConfiguration<Identity
     public void Configure(EntityTypeBuilder<Identity> builder)
     {
         builder.HasKey(x => x.Address);
-
         builder.Property(x => x.ClientId).HasMaxLength(200);
+        builder.HasOne<Tier>().WithMany(x => x.Identities).HasForeignKey(x => x.TierId).IsRequired(false);
```

Comment:
The reason why it isn't required is that it is null when migrating from an old version, right? If that's the case, could you please add a comment that explains this?